site.yml

```diff
@@ -347,3 +347,13 @@
       tags: ['openstack', 'setup', 'openstack-network']
       when: neutron.enable_external_interface|default('false')|bool
   environment: "{{ env_vars|default({}) }}"
+
+# This is the last role because ceph health checks can take up to 2 hours
+- name: ceph update
+  hosts: ceph_monitors:ceph_osds:controller:compute
```

Comment:
service restarts on controller and compute is not needed. ceph.conf  file update would be sufficient.